modules/@apostrophecms/login/index.js

```diff
@@ -158,6 +158,36 @@ module.exports = {
             await destroySession();
           }
         },
+        async setLocale(req) {
```

Comment:
I see your point, but it's debatable, and I really don't want to open this can of worms at this time. I have some thoughts on the possibility of a sessions module, separate from the express module which you wouldn't want this route to be in either, but every time I rearrange the middleware things get complicated fast. Meanwhile "this route modifies your login" is not an unreasonable statement, so I'd like to leave this as-is at least for alpha 3. This mechanism is mainly for us and our in-context, on-page admin interface experience, REST API developers would not need to rely on the session.